api/generate.go

```diff
@@ -1,4 +1,4 @@
-package gqlgen
+package api
```

Comment:
I started by moving this into `codegen`, but it caused an import loop between `plugin` and `codegen`.  I've moved it here for now, do you have another suggestion?